components/cam/src/physics/cam/phys_grid.F90

```diff
@@ -1221,55 +1314,50 @@ subroutine phys_grid_init( )
         process_ncols(owner_p) = process_ncols(owner_p) + chunk_ncols
       enddo
 
-      min_process_nthreads = npthreads(0)
-      max_process_nthreads = npthreads(0)
-      min_process_nchunks  = npchunks(0)
-      max_process_nchunks  = npchunks(0)
-      min_process_ncols    = process_ncols(0)
-      max_process_ncols    = process_ncols(0)
-      do p=1,npes-1
-        if (npthreads(p) < min_process_nthreads) &
-          min_process_nthreads = npthreads(p)
-        if (npthreads(p) > max_process_nthreads) &
-          max_process_nthreads = npthreads(p)
-
-        if (npchunks(p) < min_process_nchunks) &
-          min_process_nchunks = npchunks(p)
-        if (npchunks(p) > max_process_nchunks) &
-          max_process_nchunks = npchunks(p)
-
-        if (process_ncols(p) < min_process_ncols) &
-          min_process_ncols = process_ncols(p)
-        if (process_ncols(p) > max_process_ncols) &
-          max_process_ncols = process_ncols(p)
-      enddo
+      min_process_nthreads = minval(npthreads)
+      max_process_nthreads = maxval(npthreads)
+      min_process_nchunks  = minval(npchunks)
+      max_process_nchunks  = maxval(npchunks)
+      min_process_ncols    = minval(process_ncols)
+      max_process_ncols    = maxval(process_ncols)
+      min_pcols            = minval(pcols_proc)
+     !max_pcols            = maxval(pcols_proc)    already calculated
```

Comment:
I would take out this commented line since it's not needed. If you want to leave a note about this was calculated above then just do it in a normal comment block.

Reply:
Will do. Thanks.

Reply:
@whannah1: Done - just deleted the line. Has been pushed to the branch.